pyvista/plotting/plotting.py

```diff
@@ -6270,10 +6248,6 @@ def on_timer(iren, event_id):
         else:
             self.window_size = window_size
 
-        # add timer event callback to break out of blocking interactive update call (only needed for VTK<9)
-        if not self.iren.can_process_events:
-            self.iren.add_observer(_vtk.vtkCommand.TimerEvent, on_timer)
```

Comment:
You can also delete the `on_timer` callback now, as it's not used anywhere else as far as I know.

Reply:
Thanks. I've left a similar `on_timer` in `tests/test_render_window_interactor.py` alone, as it seems to be different.